pydicom/dataset.py

```diff
@@ -636,6 +640,24 @@ def __iter__(self):
         for tag in taglist:
             yield self[tag]
 
+    def raw(self):
+        """Iterate through the top-level of the Dataset, yielding DataElements
+        or RawDataElements (no convertion done).
```

Comment:
conversion

Reply:
Tests coverage for `raw`, too. 

Should include some tests to show that RawDataElements aren't converted.

Reply:
> This function name isn't really descriptive

This was a case of "I don't have a good name now, think about it later...". I'm not sure about this one - would `raw_iter` a better name?
> Should include some tests to show that RawDataElements aren't converted.

I will add a test.

Reply:
I'm not sure either, it doesn't only iterate raw...

Reply:
Ok, renamed to `elements()`.